crates/rolldown/tests/esbuild/default/import_forms_with_minify_identifiers_and_no_bundle/artifacts.snap

```diff
@@ -117,15 +117,15 @@ snapshot_kind: text
 ## entry.js
 
 ```js
-import * as ns2 from "foo";
 import * as ns from "foo";
-import def, { a, a2, b, default as def2, default as def3 } from "foo";
+import * as ns2 from "foo";
```

Comment:
the order just follows the orignal span of the specifier, https://github.com/rolldown/rolldown/blob/8cf601e7c4ed0589c4ffaa63c8276afccd22a2ed/crates/rolldown/tests/esbuild/default/import_forms_with_minify_identifiers_and_no_bundle/entry.js?plain=1#L3-L6